### DISCUSSION TITLE
tabpane changes in 0.7.2

### DISCUSSION THREAD

**Discussion by titusfortner** [google/docsy#1745] [General]
Oof, y'all broke all my tabs with this release! 😂 

Seriously, though, is there an upgrade guide on all the changes that were made? We are using `langEqualsHeader=true` for persistence and that no longer works, so I have to change 1,000+ instances of tabpane. Need to figure what else to update. (and I still hope to get back to the 2 PRs I made a long time ago). Thanks.

**Comment by titusfortner** [comment]
Hmm, maybe I have some custom code hidden somewhere... I forget everything I know about this code every time I look away.

**Comment by titusfortner** [comment]
I did have custom code hidden! I have removed it.

`langEqualsHeader=true` still works. Thank you, sorry for doubting. 😁 
`persist=header` works, but it looks like removing it also keeps persistence? Wasn't the previous default no persistence?

**Reply by chalin** [reply]
> I did have custom code hidden! I have removed it.
> 
> `langEqualsHeader=true` still works.

👍🏻 

>`persist=header` works, but it looks like removing it also keeps persistence? Wasn't the previous default no persistence?

Right, persistence is enabled by default since 0.7.2; to disable use `persist=disabled`

**Reply by titusfortner** [reply]
Thanks. Updated this one: https://github.com/google/docsy/pull/1202

**Reply by titusfortner** [reply]
And now this seems to be working: https://github.com/google/docsy/pull/1227

**Comment by chalin** [comment]
Hey @titusfortner - I'm wrapping up tabpane updates for this year (#1641) but, FYI, I've added your PRs to the tabpane improvements issue for the next quarter (as something to look into / follow up on):

- #1746 

I'll take a closer look at your submissions as soon as I can (and hopefully I can find a way to reproduce the issues your seeing.

Also FYI, I just updated the docs and CHANGELOG so that they reflect recent updates to tabbed panes:

- #1747

**Reply by titusfortner** [reply]
I've figured out all of the issues I had with both of those PRs, both of them should be small/obvious changes now, but I understand wanting to wait to investigate. I'm going to merge them in Selenium docs and we can see if any weird issues arise. 😂

**Comment by titusfortner** [comment]
Is there a (easy) way to have `text=true` be implicit when using `%` delimiter to get markdown in a tab? Most of our implementation is moving to markdown-only since we're importing code automatically from github and want to be able to make translations for everything else.

I couldn't figure out how the markdown was being enforced just looking at the code, so it may be happening in a completely different part of the code and definitely not easy to do this, but just curious.

**Reply by chalin** [reply]
Created an issue for that, see:

- #1749
